.pre-commit-config.yaml

```diff
@@ -17,7 +17,7 @@ repos:
 
   - repo: https://github.com/astral-sh/ruff-pre-commit
     # Ruff version.
-    rev: v0.1.3
+    rev: v0.3.5
```

Comment:
Please provide short summary about new rules.
I'm not sure that we should enable all of them.

Reply:
New rules that the current code violates:
[SIM401](https://docs.astral.sh/ruff/rules/if-else-block-instead-of-dict-get/)
[E721](https://docs.astral.sh/ruff/rules/type-comparison/)
[E266](https://docs.astral.sh/ruff/rules/multiple-leading-hashes-for-block-comment/)
[SIM113](https://docs.astral.sh/ruff/rules/enumerate-for-loop/)
[E262](https://docs.astral.sh/ruff/rules/no-space-after-inline-comment/)
[F841](https://docs.astral.sh/ruff/rules/unused-variable/)
[SIM910](https://docs.astral.sh/ruff/rules/dict-get-with-none-default/)
[E251](https://docs.astral.sh/ruff/rules/unexpected-spaces-around-keyword-parameter-equals/)
[E201](https://docs.astral.sh/ruff/rules/whitespace-after-open-bracket/)
[E231](https://docs.astral.sh/ruff/rules/missing-whitespace/)
[E203](https://docs.astral.sh/ruff/rules/whitespace-before-punctuation/)
[SIM103](https://docs.astral.sh/ruff/rules/needless-bool/)
[E265](https://docs.astral.sh/ruff/rules/no-space-after-block-comment/)
All of them look applicable. Probably, only E265 is discussable.